arcgis-ios-sdk-samples/Maps/Display device location/DisplayLocationSettingsViewController.swift

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DisplayLocationSettingsViewController: UITableViewController {
+    
+    //MARK: - Outlets
+    
+    @IBOutlet weak var showSwitch: UISwitch!
+    @IBOutlet weak var autoPanModeCell: UITableViewCell!
+    
+    //MARK: - Model
+    
+    /// The SDK object for displaying the device location. Attached to the map view.
+    weak var locationDisplay: AGSLocationDisplay?
```

Comment:
What happens if the location display is set after the view is loaded?

I realize that probably isn't happening now, but it could happen in the future if the sample were updated. Since a view controller has no control over when its own view will be loaded, it is a good idea (and a good habit to form) for any view controller to handle the model being set both before and after the view is loaded.

Reply:
This pattern has been repeated in many samples now, and while I don't mind the safety, I'm less inclined to add code that will never be needed. It the sample is updated in the future, the developer can make the change then.

Reply:
I made the change.